.env.example

```
@@ -0,0 +1 @@
+DB_PASSWORD = 
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)